arm-redis/2016-04-01/examples/RedisCacheForceReboot.json

```diff
@@ -12,6 +12,7 @@
   "responses": {
     "200": {
       "body": {
+        "Message": "reboot operation enqueued"
```

Comment:
@vishrutshah my take, revised: it should never have been PascalCase but now that it is, it can't be fixed without it being breaking, so its stuck that way til next api-version.